### DISCUSSION TITLE
Valaxy怎样自定义字体cdn？

### DISCUSSION THREAD

**Discussion by future382** [YunYouJun/valaxy#399] [Q&A] [answered]
众所周知，Google Fonts和googleapis.com在国内几乎不可用，在内地无法正常加载其字体，请问怎样才能修改Valaxy所引用的字体源？
请大佬帮忙解答，感激不尽。

**Comment by YunYouJun** [comment, marked as answer]
自定义字体参见：https://valaxy.site/guide/custom/styles#custom-font
